### PR TITLE
simplifying the new extension search endpoint a bit + shorter query parameter names

### PR DESCRIPTION
**Pull request by 2is10**
@leogrim

### CODE REVIEW

File: kifi-backend/modules/search/conf/search.routes [hunk 1]
```diff
@@ -7,7 +7,7 @@ GET         /admin/search/instance         @com.keepit.controllers.search.Search
 # Search Extension API
 ##########################################
 GET         /search        @com.keepit.controllers.ext.ExtSearchController.search(q: String, f: Option[String], maxHits: Int, lastUUID: Option[String], context: Option[String], kifiVersion: Option[KifiExtVersion] ?= None, start: Option[String], end: Option[String], tz: Option[String], coll: Option[String], debug: Option[String])
-GET         /ext/search           @com.keepit.controllers.ext.ExtSearchController.search2(q: String, f: Option[String], maxHits: Int, lastUUID: Option[String], context: Option[String], kifiVersion: Option[KifiExtVersion] ?= None, debug: Option[String])
+GET         /ext/search                    @com.keepit.controllers.ext.ExtSearchController.search2(v: KifiExtVersion, q: String, n: Int, f: Option[String], u: Option[String], c: Option[String], debug: Option[String])
```
Comment (2is10) [inline kifi-backend/modules/search/conf/search.routes:10]: I moved the required parameters to the front and shortened a few names to one letter:
`v` ← `kifiVersion`
`u` ← `lastUUID`
`c` ← `context`

**Comment by leogrim** [comment]
I merged this one before the older one => merge conflict => reverted this one => more merge conflict. I'll fix this first thing tomorrow morning - I'm tired and may just make another mistake.

**Comment by 2is10** [comment]
Don’t make me write [Mail Goggles](http://arstechnica.com/information-technology/2008/10/mail-goggles-a-breathlyzer-test-for-your-gmail/) for Github.

**Comment by leogrim** [comment]
Come on it was Wednesday :)

**Pull request merged**